x-pack/legacy/plugins/siem/public/containers/tls/index.tsx

```diff
@@ -74,29 +75,30 @@ class TlsComponentQuery extends QueryTemplatePaginated<
       startDate,
       tlsSortField,
     } = this.props;
+    const variables: GetTlsQuery.Variables = {
+      defaultIndex: chrome.getUiSettingsClient().get(DEFAULT_INDEX_KEY),
+      filterQuery: createFilter(filterQuery),
+      flowTarget,
+      inspect: isInspected,
+      ip,
+      pagination: generateTablePaginationOptions(activePage, limit),
+      sort: tlsSortField,
+      sourceId,
+      timerange: {
+        interval: '12h',
+        from: startDate ? startDate : 0,
+        to: endDate ? endDate : Date.now(),
```

Comment:
I think this is a better pattern than the others above and in the other parts of the code base where we deactivate the type checking using `startDate!` and `endDate!`.

At another time, I would re-factor the others to use this pattern.